model/histogram/float_histogram.go

```diff
@@ -30,6 +30,8 @@ import (
 type FloatHistogram struct {
 	// Counter reset information.
 	CounterResetHint CounterResetHint
+	// KahanSumHint runs compensation for lost low-order bits.
+	KahanSumHint float64
```

Comment:
I don't think we should add a field to every single histogram. It takes another 8 bytes for every histogram, including those that never need the summation.

Also, just one Kahan hint isn't enough, see below.